api/src/test/java/org/openmrs/module/ModuleClassLoaderTest.java

```diff
@@ -29,15 +30,16 @@
 	@Before
 	public void before() {
 		mockModule = new Module("mockmodule", "mockmodule", "org.openmrs.module.mockmodule", "author", "description", "1.0");
-		mockModules = new HashMap<String, String>();
+		mockModules = new HashMap<>();
 	}
 	
 	/**
-	 * @verifies return true if file matches and openmrs version matches
```

Comment:
I see more changes in this class than the original commit. Did you intentionally make these extra changes?

Reply:
@dkayiwa true, at least another commit that did not happen yet on 2.1.x is affecting this test file and is coming now through cherry-picking [e045948](https://github.com/openmrs/openmrs-core/commit/e04594822fb19117fe665623cf9b9b11b4834dba). The file would become exactly as it is on master.

Reply:
So are you reverting those extra changes?

Reply:
Why would we do that since this brings 2.1.x a more in line with 2.2 (master)?

Reply:
When back porting, we usually do not do more than the original commit. If there are any improvements to the old branch, we do that using a new pull request. In other words, the changes we back port should be exactly like the original.

Reply:
Ok so... I did this on my feature branch:
```
git checkout --patch upstream/master api/src/test/java/org/openmrs/module/ModuleClassLoaderTest.java
```
That let me interactively pick only the master changes that are relevant to the original feature.